extension/test/integration/make-payment.handler.spec.js

```diff
@@ -1,97 +1,122 @@
 const { expect } = require('chai')
 
 const ctpClientBuilder = require('../../src/ctp')
-const iTSetUp = require('./integration-test-set-up')
-const constants = require('../../src/config/constants')
 const config = require('../../src/config/config')
+const constants = require('../../src/config/constants')
+const iTSetUp = require('./integration-test-set-up')
+
+describe('::make-payment with multiple adyen accounts use case::', () => {
```

Comment:
Shouldn't we also add a scenario of with multiple CTP projects but one has incorrect credentials?

Reply:
using integration tests for obvious failures has no benefit in general, we might have a unit test for such cases? Both in config and validation cases are unit tests. @lojzatran could you double check.

Reply:
I don't think it's necessary to use integration tests for error cases. We already have some unit tests here.